src/terragraph-e2e/lua/tg/frr_utils.lua

```diff
@@ -295,17 +306,32 @@ function frr_utils.fillConfigTemplate(
 
   -- Fill in neighbors
   local neighborsInfo, afNeighborsInfo, activeNeighbors = {}, {}, {}
+  local neighborMd5Password = ""
+  local neighborGeneralConfig = ""
   for k, v in tablex.sort(neighbors or {}) do
-    neighborsInfo[#neighborsInfo + 1] = NEIGHBOR_INFO_FORMAT:substitute{
+
+    if staticBgpParams.md5Password == "" then
```

Comment:
to be safe, I would write `if staticBgpParams.md5Password == nil or staticBgpParams.md5Password == ""`

Reply:
value=nil handled at line 185.